CONTRIBUTING.md

```diff
@@ -0,0 +1,291 @@
+
+##### Added to changelog
+
+- `fix:` Patches a bug in the codebase. Nothing new is introduced in terms of functionality.
+- `feat:` Introduces a new feature to the codebase. A new component is an often use case.
+- `BREAKING-CHANGE:` Introduces a breaking change to existing functionality.
+  - **Examples:**
+    - A component is removed from a package
+    - Functionality of a component is changed in a way that requires the end user to perform an action
+
+##### Not added to changelog
+
+- `build:` Changes that affect the build system or external dependencies (example scopes: rollup, stylelint, npm)
+- `chore:` Other changes that don't modify src or test files
+- `docs:` Documentation only changes
+- `style:` Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- `test:` Adding missing tests or correcting existing tests
+- `refactor:` A code change that neither fixes a bug nor adds a feature
+- `revert:` Reverts a previous commit
+- `perf:` A code change that improves performance
+
+---
+
+### How to write and structure your code
+
+To ensure a consistent and enjoyable coding experience for everyone, we have established guidelines for writing our code.
+
+#### Styling with CSS Modules
+
+We use CSS modules to style our components. This prevents naming conflicts by adding a unique prefix to all components.
+A CSS module file is created by adding `.module.css` to the end of the CSS file.
+
+#### Use of design tokens
+
+When styling our components we try to always use semantic tokens from the `@digdir/design-system-tokens` package when available.
+Using hard-coded values is not reusable and we therefore try to avoid this.
+Take a look at how other components are styled to see examples.
+
+#### Code formatting with Prettier
+
+In this project, we employ [Prettier](https://prettier.io/) for code formatting. It is advisable to configure your code editor to automatically format files upon saving. This practice will prove beneficial when merging your changes into the main branch. It's worth noting that we enforce rigorous code checks in pull requests, emphasizing the importance of consistent code formatting.
+
+#### Linting with ESLint and Stylelint
+
+TypeScript and CSS files have been configured with linting, which means that the project will scan these files for potential problems or issues. Linting helps maintain code quality by detecting errors, enforcing coding conventions, and promoting best practices. You have to fix all errors and warnings before the code can be merged into the main branch.
```

Comment:
Maybe we merge these chapters? Prettier and eslint are kinda intertwined. https://prettier.io/docs/en/integrating-with-linters.html

We also use [Editorconfig](https://editorconfig.org/) for defining rules/formatting for IDE.

Reply:
Merged chapters